src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/ShardIndexingPressureMetricsCollector.java

```diff
@@ -0,0 +1,211 @@
+                                .append(PerformanceAnalyzerMetrics.sMetricNewLineDelimitor);
+                            value.append(new ShardIndexingPressureStatus(AllMetrics.IndexingStage.COORDINATING.toString(),
+                                shardId.get("indexName").toString(), shardId.get("id").toString(),
+                                Long.parseLong(tracker.get("coordinatingRejections").toString()),
+                                Long.parseLong(tracker.get("currentCoordinatingBytes").toString()),
+                                Long.parseLong(tracker.get("primaryAndCoordinatingLimits").toString()),
+                                Double.longBitsToDouble(Long.parseLong(tracker.get("coordinatingThroughputMovingAverage").toString())),
+                                Long.parseLong(tracker.get("lastSuccessfulCoordinatingRequestTimestamp").toString())).serialize())
+                                .append(PerformanceAnalyzerMetrics.sMetricNewLineDelimitor);
+                            value.append(new ShardIndexingPressureStatus(AllMetrics.IndexingStage.PRIMARY.toString(),
+                                shardId.get("indexName").toString(), shardId.get("id").toString(),
+                                Long.parseLong(tracker.get("primaryRejections").toString()),
+                                Long.parseLong(tracker.get("currentPrimaryBytes").toString()),
+                                Long.parseLong(tracker.get("primaryAndCoordinatingLimits").toString()),
+                                Double.longBitsToDouble(Long.parseLong(tracker.get("primaryThroughputMovingAverage").toString())),
+                                Long.parseLong(tracker.get("lastSuccessfulPrimaryRequestTimestamp").toString())).serialize())
+                                .append(PerformanceAnalyzerMetrics.sMetricNewLineDelimitor);
+                            value.append(new ShardIndexingPressureStatus(AllMetrics.IndexingStage.REPLICA.toString(),
+                                shardId.get("indexName").toString(), shardId.get("id").toString(),
+                                Long.parseLong(tracker.get("replicaRejections").toString()),
+                                Long.parseLong(tracker.get("currentReplicaBytes").toString()),
+                                Long.parseLong(tracker.get("replicaLimits").toString()),
+                                Double.longBitsToDouble(Long.parseLong(tracker.get("replicaThroughputMovingAverage").toString())),
+                                Long.parseLong(tracker.get("lastSuccessfulReplicaRequestTimestamp").toString())).serialize())
+                                .append(PerformanceAnalyzerMetrics.sMetricNewLineDelimitor);
+                        } catch (JsonProcessingException | ParseException e) {
+                            LOG.debug("Exception raised while parsing string to json object. Skipping IndexingPressureMetricsCollector");
```

Comment:
The default log level is not debug. So, how would we know if this exception is generated often ?